src/fe-small-advertisements/src/App.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function App() {
+  return (
+    <>
+  <h1>Hello World</h1>
```

Comment:
Perhaps you want to add some spaces here for the `h1` to be aligned better

Reply:
Of course, I didn't know Prettier wouldn't work on .tsx files by default and I didn't pay attention, my mistake. I already found how to fix it. I hope it works.

Reply:
For now just adding some tabs can work. We will set up prettier and eslint in a future PR.

Reply:
Yes. Setting the Prettier and the eslint is a different issue on the board.